conf/messages.en

```diff
@@ -396,6 +396,21 @@ summary.contact-details.contactName=Full name
 summary.contact-details.contactEmail=Email address
 summary.contact-details.contactNumber=Phone number
 
+# File upload messages
+# ----------------------------------------------------------
+view.upload-file.first.title=Upload your first document
+view.upload-file.first.heading=Upload your first document
+view.upload-file.next.title=Upload another document
+view.upload-file.next.heading=Upload another document
+view.upload-file.waiting=We are checking your file, please wait
+view.file-uploaded.title=You have uploaded {0} document
```

Comment:
(You are probably well aware but just in case it wasn't noticed I'll mention that this will need to be plural after first upload)